source/nodejs/adaptivecards-designer/README.md

```diff
@@ -16,18 +16,28 @@ There are two simple ways to consume the designer: CDN script reference or impor
 
 ### Option 1: CDN script references
 
-The simplest way to get started it to include 3 script tags in your page. 
+The simplest way to get started it to include 6 script tags in your page. 
 
 1. **monaco-editor** - provides a rich JSON-editing experience
-2. **markdown-it** - [optional] automatic markdown support for the designer and cards
+2. **adaptivecards** - adaptive card library used by designer
+3. **adaptivecards-templating** - provides data binding support for designer
+4. **adaptive-expressions** - used by templating library to enable daya binding support
+5. **markdown-it** - [optional] automatic markdown support for the designer and cards
 
 To load the designer component you have 2 options:
 
-3. 
+6. 
    * **adaptivecards-designer** - the designer component, with default Microsoft hosts included (Teams, Outlook, Cortana, etc)
    * **adaptivecards-designer-standalone** - the standalone designer component, without any standard Host Config containers
 
 ```html
+<!-- REQUIRED: adaptivecards library for designer to work -->
+<script src="https://unpkg.com/adaptivecards@latest/dist/adaptivecards.min.js"></script>
+<!-- REQUIRED: adaptive-expressions is required by adaptivecards-templating library to enable data binding support in designer -->
+<script src="https://unpkg.com/adaptive-expressions@4.11.0-dev.20201027.ca35ce4/lib/browser.js"></script>
```

Comment:
Is the hardcoded version number in here required? Can we change to @latest or some format that pulls the latest in 4.x in case any breaking changes are introduced?

Reply:
changed to rc version